scripts/dashboard/TransferMenu.vue

```diff
@@ -146,6 +146,8 @@ async function selectContact() {
                             type="number"
                             placeholder="0.00"
                             autocomplete="nope"
+                            onkeypress="return event.charCode >= 46 && event.charCode <= 57"
+                            inputmode="numeric"
```

Comment:
```suggestion
                            inputmode="decimal"
```

Change the inputmode to `decimal`. Using `numeric` forces users to only send integer amounts of PIV on mobile devices, since the virtual keyboard adapts to this mode, example below.

https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/inputmode#decimal

![image](https://github.com/user-attachments/assets/0b6b34fe-3d77-479a-88e9-6981e4017f1e)